Landing page

vickyme16 opened the pull request:
https://vickyme16.github.io/layout_landing-page

volodymyr-soltys97 reviewed (changes requested):
Good job!
Let's improve your page
1. Add topical favicon and change title on the page 
<img width="171" alt="image" src="https://github.com/user-attachments/assets/ec70ffa8-86bf-4888-a055-c8ed9af1b149">

2. You need to show additional phone number when hovering this icon 
<img width="82" alt="image" src="https://github.com/user-attachments/assets/7f50a5f5-ae4c-43ee-bb55-94eb43f37ad4">

3. Remove scroll when the menu is open 
<img width="1508" alt="image" src="https://github.com/user-attachments/assets/960512a4-1a50-44ab-8d09-93d187989492">

4. Fix size for the logo 
<img width="274" alt="image" src="https://github.com/user-attachments/assets/6066e664-73af-4e4b-b3ef-6f604128408a">

5. Fix font weight for the prices 
<img width="1105" alt="image" src="https://github.com/user-attachments/assets/0160dea7-0663-486f-a217-b44844eadc8c">

6. The images shouldn't overlap each other when hovering 
<img width="1269" alt="image" src="https://github.com/user-attachments/assets/085ed4b7-e4fb-41cb-961b-5c879742c053">

7. Add some hover effects for all clickable elements 
<img width="273" alt="image" src="https://github.com/user-attachments/assets/1a97e452-6ac0-4f24-b475-eb79bed1ef7a">

vickyme16 commented:
Hello, 
I fixed 2, 4-7 tasks, and I have questions about 1 and 3 tasks:
1. Add topical favicon and change title on the page - where can I find the correct favicon?
3. Remove scroll when the menu is open - I added z-index: 10, but the scroll is still there, not sure how to remove it



> Good job! Let's improve your page
> 
> 1. Add topical favicon and change title on the page
> 
> <img alt="image" width="171" src="https://private-user-images.githubusercontent.com/64750958/382598841-ec70ffa8-86bf-4888-a055-c8ed9af1b149.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA2NjI3NzgsIm5iZiI6MTczMDY2MjQ3OCwicGF0aCI6Ii82NDc1MDk1OC8zODI1OTg4NDEtZWM3MGZmYTgtODZiZi00ODg4LWEwNTUtYzhlZDlhZjFiMTQ5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTAzVDE5MzQzOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWRmYjUzN2Q2NzM5YTYzOGMyMzMzNjg5MzU3YzYwMGE3YmJlOGMzYWNmYzkyMjA4MDEwYzk2ZjY1ZjU4YjkzOTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.z7pgUJfehcuEsV454hqbcKbAwhSnFiZpWBfc8j3xl7w">
> 2. You need to show additional phone number when hovering this icon
> 
> <img alt="image" width="82" src="https://private-user-images.githubusercontent.com/64750958/382598881-7f50a5f5-ae4c-43ee-bb55-94eb43f37ad4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA2NjI3NzgsIm5iZiI6MTczMDY2MjQ3OCwicGF0aCI6Ii82NDc1MDk1OC8zODI1OTg4ODEtN2Y1MGE1ZjUtYWU0Yy00M2VlLWJiNTUtOTRlYjQzZjM3YWQ0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTAzVDE5MzQzOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY0MTY2ZTZhMTlhYmMwMTY1N2RlN2EwZWY3ZmY0MTFjMWZhMmI5MDU3YzhmOWY2MGUwMGM5ZDcwZTAxMzljZGMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.t4GUdBUK_M_9-iSRgVZa7710aQtYURtMEi1Bm7PY1I8">
> 3. Remove scroll when the menu is open
> 
> <img alt="image" width="1508" src="https://private-user-images.githubusercontent.com/64750958/382598897-960512a4-1a50-44ab-8d09-93d187989492.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA2NjI3NzgsIm5iZiI6MTczMDY2MjQ3OCwicGF0aCI6Ii82NDc1MDk1OC8zODI1OTg4OTctOTYwNTEyYTQtMWE1MC00NGFiLThkMDktOTNkMTg3OTg5NDkyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTAzVDE5MzQzOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRmZjdhNDIxYjA1OWNhODBhMjc1Y2RhN2NmMmJmMGE1ZmM3N2M3OTdiMjAxNmRlOGEyZjk2YjUwMzExNTJiN2QmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.pLou_my2MmqbgslpHtTld0ZrOQ1ae8D9cZf2MZmllaw">
> 4. Fix size for the logo
> 
> <img alt="image" width="274" src="https://private-user-images.githubusercontent.com/64750958/382598913-6066e664-73af-4e4b-b3ef-6f604128408a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA2NjI3NzgsIm5iZiI6MTczMDY2MjQ3OCwicGF0aCI6Ii82NDc1MDk1OC8zODI1OTg5MTMtNjA2NmU2NjQtNzNhZi00ZTRiLWIzZWYtNmY2MDQxMjg0MDhhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTAzVDE5MzQzOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdiYThjNzMyMmVlNDIyM2VhY2JkZTczYjY0OTQyN2ExMmU4NGZiMWNjNDZkODg0N2Q2ZDNhNGI2MDRjZDI0ZGEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.0aKJhomamLiyj11vaJRJ3OVoc9nXhsl5kds5gnrEGH8">
> 5. Fix font weight for the prices
> 
> <img alt="image" width="1105" src="https://private-user-images.githubusercontent.com/64750958/382598941-0160dea7-0663-486f-a217-b44844eadc8c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA2NjI3NzgsIm5iZiI6MTczMDY2MjQ3OCwicGF0aCI6Ii82NDc1MDk1OC8zODI1OTg5NDEtMDE2MGRlYTctMDY2My00ODZmLWEyMTctYjQ0ODQ0ZWFkYzhjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTAzVDE5MzQzOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWRkMDI4N2EyMjY5YTNhMWVhZDk2YzA3MjNjM2I5MTYzZGJhODJjMTY0MmNiYjY5OTAyYTUwNTA2YTc4ODc2MWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.HxFZRYcMneT4-ZsJngxC446O6uXkPUQMA0NTf2oEU10">
> 6. The images shouldn't overlap each other when hovering
> 
> <img alt="image" width="1269" src="https://private-user-images.githubusercontent.com/64750958/382598972-085ed4b7-e4fb-41cb-961b-5c879742c053.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA2NjI3NzgsIm5iZiI6MTczMDY2MjQ3OCwicGF0aCI6Ii82NDc1MDk1OC8zODI1OTg5NzItMDg1ZWQ0YjctZTRmYi00MWNiLTk2MWItNWM4Nzk3NDJjMDUzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTAzVDE5MzQzOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFmZGU1ZjY4ZjNjMzlhY2ZhMDM3MjRiNzQyNGZjMzI3OTBkNWIwMTEwOGRkM2QzYjA4ZWQ4ODBkZjU2ZTlkNjAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.xfWty0wOMY3Zy_lWvTOADhV3kA39AikQ44S4BekHXG8">
> 7. Add some hover effects for all clickable elements
> 
> <img alt="image" width="273" src="https://private-user-images.githubusercontent.com/64750958/382599024-1a97e452-6ac0-4f24-b475-eb79bed1ef7a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA2NjI3NzgsIm5iZiI6MTczMDY2MjQ3OCwicGF0aCI6Ii82NDc1MDk1OC8zODI1OTkwMjQtMWE5N2U0NTItNmFjMC00ZjI0LWI0NzUtZWI3OWJlZDFlZjdhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTAzVDE5MzQzOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg1YWIzOWZjMTkzNTRiNTQ4NGY0MzVjODBmZDdiNjA1YTc0YTUyYTc5MzdlNzc3YmU2NzJmNGVhNjhlMGE3NTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.qRh-Yg8FA2apKZe2ZqsbrzU8x-uAMxVf2AsAb_hHcJM">
